internal/karma/karma.conf.js

```diff
@@ -49,8 +103,8 @@ module.exports = function(config) {
 
     // base path that will be used to resolve all patterns (eg. files, exclude)
     basePath: 'TMPL_runfiles_path',
-    files: [
-TMPL_files
-    ]
+
+    // files
```

Comment:
stray comment?

Reply:
yeah. a bit redundant. there was a comment before every other property and I didn't know what else to put.